pkg/api/steve/apiaccess/apiaccess.go

```diff
@@ -1,4 +1,4 @@
-package disallow
+package apiaccess
```

Comment:
1. I prefer not to change the name of public packages when we are not changing the behavior. Though this package is not imported by many others, so it should not be a big deal.

2. If we are set on changing the name, we shouldn't use `apiaccess` since this only handles deciding which CRDs are exposed by `steve`. There are other parts to the Steve API and Rancher's API that this package does not handle, We are also not checking an individual user's access in this package.

Reply:
Sounds good, will drop the name-changing commit.